Default (OSX).sublime-keymap

```diff
@@ -0,0 +1,4 @@
+[{
```

Comment:
You may want to consult the list of default keybindings to make sure yours isn’t taken. This is located on the menu bar:
(OSX) “Sublime Text” >  “Preferences” > “Key Bindings – Default”
(Windows) “Preferences” > “Key Bindings – Default”